test/cppsuite/test_harness/workload/database_model.h

```diff
@@ -30,37 +30,26 @@
 #define DATABASE_MODEL_H
 
 #include <atomic>
-#include <map>
-#include <chrono>
 #include <string>
-#include <tuple>
+#include <map>
 
 #include "../timestamp_manager.h"
```

Comment:
You can use forward class declaration instead of the include

Reply:
What's the advantage of forward declaration over include in this instance?